### ISSUE TITLE
Duplicate android devices

### ISSUE THREAD

**Issue by rosscoau** [jacksonjesse/covidsafe-scanner#1]
Hi,

To help you with duplicate android devices.. I'm seeing in the logs for an android device that the peripheral identifier is changing but the msg in value is the same - so I suspect you can match devices based on the msg???

HTH

rossco

**Comment by jacksonjesse** [comment]
Hey there!

It would be super helpful if you could post the logs (anything you don’t want seen can be de-identified, though this data isn’t particularly identifying).

Are you referring to the “msg” that comes in the JSON payload? I haven’t done too much reading about what that represents but you may be right that it doesn’t change, so could try using that as a key.

I committed some code a couple of days ago to add some extra logs so that would help heaps. If that doesn’t help me spot the best way to sort it I can add even more logging and get you to try again.

**Comment by jacksonjesse** [comment]
On second thoughts, it may be best to avoid logs in case there is anything in there that would be best not made public. I’ll look further into this but if there is any further info you can provide outside of logs that would be welcome.

**Comment by rosscoau** [comment]
Yes the msg in the payload is unchanging for a while.  I think the msg has changed today.  I'll do some more analysis on the debug output and follow up.  I suspect the msg will change periodically and this is stored on the server against your account and used to match to your device on upload if/when someone else uploads their data for tracing.
